arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1115,91 +1146,88 @@
           "type": "array",
           "items": {
             "type": "string"
           }
         },
         "path": {
-          "description": "The eventhub path.",
-          "type": "string"
+          "description": "The Event Hub-compatible name.",
+          "type": "string",
+          "readOnly": true
         },
         "endpoint": {
-          "description": "The endpoint.",
-          "type": "string"
-        },
-        "internalAuthorizationPolicies": {
-          "description": "The internal authorization rules.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/SharedAccessAuthorizationRule"
-          }
-        },
-        "authorizationPolicies": {
-          "description": "The authorization rules.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/SharedAccessAuthorizationRule"
-          }
+          "description": "The Event Hub-compatible endpoint.",
+          "type": "string",
+          "readOnly": true
         }
       }
     },
     "StorageEndpointProperties": {
-      "description": "The properties of the Storage Endpoint for file upload.",
+      "description": "The properties of the Azure Storage endpoint for file upload.",
       "type": "object",
       "properties": {
         "sasTtlAsIso8601": {
-          "description": "SAS time to live. Range: 1 Min (PT1M) - 1 Day (P1D).",
+          "description": "The period of time for which the the SAS URI generated by IoT Hub for file upload is valid. See: https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-file-upload#file-upload-notification-configuration-options.",
           "type": "string",
           "format": "duration"
         },
         "connectionString": {
-          "description": "The account key credentials for storage account selected by customer for uploading files.",
+          "description": "The connection string for the Azure Storage account to which files are uploaded.",
           "type": "string"
         },
         "containerName": {
-          "description": "The root container name where all files will be uploaded.",
+          "description": "The name of the root container where you upload files. The container need not exist but should be creatable using the connectionString specified.",
           "type": "string"
         }
-      }
+      },
+      "required": [
+        "connectionString",
+        "containerName"
+      ]
     },
     "MessagingEndpointProperties": {
-      "description": "The properties of the Messaging Endpoints used by this IoT Hub.",
+      "description": "The properties of the messaging endpoints used by this IoT hub.",
       "type": "object",
       "properties": {
         "lockDurationAsIso8601": {
-          "description": "The lock duration. Range: 5 Sec (PT5S) - 5 Min (PT5M).",
+          "description": "The lock duration. See: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-file-upload.",
```

Comment:
The doc in the url says that the default is 60 seconds. Please update the swagger spec accordingly.